bin/www

```diff
@@ -16,7 +16,7 @@ const app = require('../app');
  * Get port from environment and store in Express.
  */
 
-const port = (process.env.NODE_ENV === 'production') ? process.env.PORT : 3000;
+const port = (process.env.NODE_ENV === 'production') ? process.env.PORT : 3001;
```

Comment:
Ok tbh I forgot to change this back but how about if we leave this? Layers-api uses 3000 locally as well by default and it's handy to have them be different when you need to point to your local version of both.